controllers/horizon_controller.go

```diff
@@ -907,6 +907,10 @@ func (r *HorizonReconciler) generateServiceConfigMaps(
 		templateParameters["SSLCertificateKeyFile"] = fmt.Sprintf("/etc/pki/tls/private/%s.key", horizon.ServiceName)
 	}
 
+	templateParameters := map[string]interface{}{
+		"LogFile": horizon.LogFile,
+	}
```

Comment:
Oh, this will override the entire templateParameters and you will just end up with LogFile in there. So you want to just set the `LogFile` key:

```suggestion
	templateParameters["LogFile"] = horizon.LogFile
```

Reply:
Code working as is - if this causes an issue, we can revisit.